### DISCUSSION TITLE
How to integrate Bullhorn Job Search into Wordpress

### DISCUSSION THREAD

**Discussion by msovan** [bullhorn/career-portal#513] [Q&A]
Hi,

I was wondering if someone could please guide me in the right direction with integrating Bullhorn. Currently our website displays the following:

![image](https://user-images.githubusercontent.com/130591580/231531830-2eed0ada-7b79-4aba-a704-996caea032eb.png)

But jobs are not posting. Please advise!

**Comment by pbearne** [comment]
Hi

Have you published any jobs?
Here is the instruction from the Matador Jobs plugin https://docs.matadorjobs.com/articles/published-status-to-import/. The Bullhorn part is the same.

If you what to use the WordPress search then you might need to look at Matador Jobs as it integrates directly into WordPress.

paul@matadorjobs.com

**Comment by msovan** [comment]
Hi,

Yes, we have a few published jobs. The jobs state “published on career portal” in Bullhorn.

***@***.***

Bophamary “Mary” Sovan  [LinkedIn icon] <http://www.linkedin.com/in/bophamarysovan>
Business Analyst
P: 714.316.1840 | E: ***@***.******@***.***>
485 E. 17th Street, Suite 550, Costa Mesa, CA 92627
www.globalitresourcesinc.com<http://www.globalitresourcesinc.com/>

**Comment by pbearne** [comment]
Check if they are open / not closed.

**Reply by msovan** [reply]
They are open jobs, it looks like the plug-in stopped working as of last year.

https://wordpress.org/plugins/bullhorn-oscp/

**Reply by pbearne** [reply]
Then check out Matador Jobs, its our replacment/upgrade for the OSCP
I belive Bullhorn no longer have a dev working on the WP plugin
This is way I am reply here to try and help
